<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addshop</class>
 <widget class="QDialog" name="addshop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>66</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true"> font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true"> font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true"> font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>门店地址</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true"> font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>联系电话</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true"> font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>经理姓名</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>160</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">
       border-style: solid; 
       background: transparent;  
       background-color:#FFFFFF; 
         


 
  font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="styleSheet">
       <string notr="true">
       border-style: solid; 
       background: transparent;  
       background-color:#FFFFFF; 
         


 
  font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="styleSheet">
       <string notr="true">
       border-style: solid; 
       background: transparent;  
       background-color:#FFFFFF; 
         


 
  font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="styleSheet">
       <string notr="true">
       border-style: solid; 
       background: transparent;  
       background-color:#FFFFFF; 
         


 
  font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="styleSheet">
       <string notr="true">
       border-style: solid; 
       background: transparent;  
       background-color:#FFFFFF; 
         


 
  font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> border-style:solid;  
       background: transparent;  
       background-color: rgb(85, 170, 255); 
         font: 12pt &quot;微软雅黑&quot;;
        
        border-radius: 10px;  
</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
